cpp/include/Ice/LocalException.h

```diff
@@ -1686,26 +1666,6 @@ namespace Ice
         void ice_print(std::ostream& stream) const override;
     };
 
-    /**
-     * This exception indicates that a response for an unknown request ID has been received.
-     * \headerfile Ice/Ice.h
-     */
-    class ICE_API UnknownRequestIdException : public ProtocolException
```

Comment:
Not used since at least Ice 3.7. An unknown request ID is ok as it corresponds to canceled request.